src/pybind/mgr/volumes/module.py

```diff
@@ -68,6 +68,12 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'desc': "Rename a CephFS volume by passing --yes-i-really-mean-it flag",
             'perm': 'rw'
         },
+        {
+            'cmd': 'fs volume info '
+                   'name=vol_name,type=CephString ',
+            'desc': "Get metadata of a volume",
```

Comment:
Can reword: `Get the information of a CephFS volume`

Reply:
Done